docs/configuration.md

```diff
@@ -258,6 +258,10 @@ Specify which authentication flow should be used when performing Microsoft authe
 
 Defaults to the value `auto`.
 
+**Note:** If [`credential.msauthUseBroker`](#credentialmsauthusebroker) is set
+to `true` and the operating system authentication broker is available, all flows
+will be delegated to the broker; this setting has no effect.
```

Comment:
By "this setting" you mean `credential.msauthFlow`, and by "has no effect" you mean that `.msauthUseBroker` overrides `.msauthFlow`, right?

Reply:
Yeah. Grammar here is hard. Any suggestions to make it clearer?

Reply:
Suggestion: truncate existing sentence at the `;`. Then add a second sentence:
> If both of those things are true, then the value of `credential.msauthFlow` has no effect.